game/core.py

```diff
@@ -63,6 +63,7 @@ def add_player(self, alias):
 
     def add_bot(self):
         if len(self.players) < 6:
+            self.num_bots+=1
```

Comment:
Why do you need this self.num_bots variable? You can create a function which counts all players with AI flag True

Reply:
Okay, yeah I've gone ahead and done that